cmd/flags.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import "github.com/urfave/cli/v2"
+
+var (
+	//ConfigPathFlag specifies the config file path.
+	ConfigPathFlag = &cli.StringFlag{
```

Comment:
move into cmd/sync and make it unexported

Reply:
api need this flag too.